README.md

```diff
@@ -264,7 +264,8 @@ semantics onto C++ such as Rust-inspired
 You can get started playing with Carbon by checking out the codebase and using
 the Carbon explorer:
 
-```shell
+```
+shell
```

Comment:
This was intentionally on the same like as the quotes, and doesn't work if it's on a separate line. See [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting) for more information.

Reply:
I agree with @geoffromer 

@lakshminarasimmanv you can see the result of this difference:
https://github.com/lakshminarasimmanv/carbon-lang/tree/patch-1#getting-started
![image](https://user-images.githubusercontent.com/23168063/181215270-d0aac619-85f9-42fc-9271-45d7a1843335.png)